src/App.tsx

```diff
@@ -15,48 +15,97 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NAME = 'name',
+  LENGTH = 'length',
+}
+
+function sortArray(
+  goods: string[],
+  howSort: SortType,
+  isReversed: boolean,
+) {
+  const array = [...goods];
+
+  array.sort((value1, value2) => {
+    if (howSort === SortType.NAME) {
+      return value1.localeCompare(value2);
+    }
+
+    if (howSort === SortType.LENGTH) {
+      return value1.length - value2.length;
+    }
+
+    return 0;
+  });
+
+  if (isReversed) {
+    array.reverse();
+  }
+
+  return array;
+}
+
 export const App: React.FC = () => {
+  const [sort, setSort] = useState('');
```

Comment:
You can add an empty string to the sort type as a default type. For example
```suggestion
  const [sort, setSort] = useState(SortType.NONE);
```